homework-3-promises-and-worker-threads/task5/index.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
+
+async function readFromFile(filename) {
+   return new Promise((resolve,reject)=>{
```

Comment:
I think we can do this without Promise, and we can use in this case readFileSync

Reply:
Fixed